bedita-app/models/behaviors/captions.php

```diff
@@ -0,0 +1,136 @@
+<?php
+/*-----8<--------------------------------------------------------------------
+ * 
+ * BEdita - a semantic content management framework
+ * 
+ * Copyright 2018 ChannelWeb Srl, Chialab Srl
+ * 
+ * This file is part of BEdita: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published 
+ * by the Free Software Foundation, either version 3 of the License, or 
+ * (at your option) any later version.
+ * BEdita is distributed WITHOUT ANY WARRANTY; without even the implied 
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details.
+ * You should have received a copy of the GNU Lesser General Public License 
+ * version 3 along with BEdita (see LICENSE.LGPL).
+ * If not, see <http://gnu.org/licenses/lgpl-3.0.html>.
+ * 
+ *------------------------------------------------------------------->8-----
+ */
+
+/**
+ * Compact find result
+ * 
+ * in model var actAs = array("CompactResult" => array('model to esclude'));
+ *
+ */
+class CaptionsBehavior extends ModelBehavior {
+
+    var $config = array();
+
+    function setup(&$model, $config = array()) {
+    }
+
+    /**
+     * Load captions in model data.
+     *
+     * @param object $model The model
+     * @param array $results Fetched models data
+     * @param bool $primary Is primary
+     * @return array
+     */
+    public function afterFind(&$model, $results, $primary)
```

Comment:
[minor] The `&` notation when parameter's value is an object is no longer needed as of PHP 5 — it's a leftover from PHP 4. Objects are always passed by reference.

Reply:
right. I got the signature from cakephp cookbook for version 1.3